src/core/Table/columns/actionColumn.tsx

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { HeaderProps } from 'react-table';
+import { Checkbox } from '../../Checkbox';
+import SvgColumnManager from '@itwin/itwinui-icons-react/cjs/icons/ColumnManager';
+import { DropdownMenu } from '../../DropdownMenu';
+import { IconButton } from '../../Buttons/IconButton';
+import { MenuItem } from '../../Menu';
+import { TABLE_RESIZE_START_ACTION } from '../actionHandlers';
+import { EXPANDER_CELL_ID, SELECTION_CELL_ID } from '../hooks';
+
+const ACTION_CELL_ID = 'iui-table-action';
+
+export const ActionColumn = <T extends Record<string, unknown>>({
+  hasColumnManager = true,
```

Comment:
So what would be the best name for this? Sounds generic even though now it gives only columns visibility

@bentleyvk @mayank99

Reply:
Is it even makes sense to have this prop? Or you think users might use several action columns?

Reply:
My suggestion `showColumnManager`. So yes, I would still prefer column manager name 😀

Reply:
I assume they could have as many as thy want.
Also, if they take action column, not necessarily they want column manager in there.
Just I thought if its column manager and later has more features (visiblity and order), how that would be set up

Reply:
I suggest we rename the current flag to `columnManager`, so that it doesn't need to be confined to a boolean.

In the future, we could accept an object as an alternative to the boolean.
```ts
type ActionColumnParams = {
  columnManager: boolean | { enableVisibilityToggle?: boolean; enableReordering?: boolean };
};
```

Usage:
```ts
ActionColumn({ columnManager: true })
```
```ts
ActionColumn({ columnManager: false }) // equivalent to ActionColumn()
```
```ts
ActionColumn({ columnManager: { enableVisibilityToggle = true, enableReordering = false } })
```